src/wpantund/NCPInstanceBase-Addresses.cpp

```diff
@@ -918,38 +918,80 @@ NCPInstanceBase::find_prefix_entry(const IPv6Prefix &prefix, const OnMeshPrefixE
 	return iter;
 }
 
+std::multimap<NCPInstanceBase::IPv6Prefix, NCPInstanceBase::OnMeshPrefixEntry>::iterator
+NCPInstanceBase::find_domain_prefix_entry(const IPv6Prefix &prefix, const OnMeshPrefixEntry &entry)
```

Comment:
We don't seem to use `entry`. Can we consider removing it from list of parameters?